monai/networks/blocks/selfattention.py

```diff
@@ -179,7 +197,9 @@ def forward(self, x):
 
             att_mat = self.drop_weights(att_mat)
             x = torch.einsum("bhxy,bhyd->bhxd", att_mat, v)
+
         x = self.out_rearrange(x)
-        x = self.out_proj(x)
+        if self.include_fc:
+            x = self.out_proj(x)
```

Comment:
Note that `out_proj` is defined even if not used, this is compatible with previously saved weights. It seems we have to keep it for this compatibility reason.

Reply:
Yes, and also we need defined it to make torchscript can be converted correctly.